src/meta/api/src/schema_api_impl.rs

```diff
@@ -1547,36 +1547,20 @@ impl<KV: kvapi::KVApi<Error = MetaError> + ?Sized> SchemaApi for KV {
 
     #[logcall::logcall]
     #[fastrace::trace]
-    async fn get_table_meta_history(
+    async fn get_gc_ready_tables(
         &self,
-        database_name: &str,
-        table_id_history: &TableIdHistoryIdent,
-    ) -> Result<Vec<(TableId, SeqV<TableMeta>)>, KVAppError> {
-        let table_name = &table_id_history.table_name;
-
-        let (_seq, table_id_list) = self.get_pb_seq_and_value(table_id_history).await?;
-
-        let table_id_list = table_id_list.ok_or_else(|| {
-            AppError::from(UnknownTable::new(
-                table_name,
-                format!("get_table_history: {}.{}", database_name, table_name),
-            ))
-        })?;
-
-        let now = Utc::now();
-
-        let metas = get_table_meta_history(self, &now, table_id_list).await?;
+        history_ident: &TableIdHistoryIdent,
+    ) -> Result<Vec<(TableId, SeqV<TableMeta>)>, MetaError> {
+        let Some(seq_table_id_list) = self.get_pb(history_ident).await? else {
```

Comment:
So this means : if a table under a db which contains 100 tables is invalid, list table from this db will return empty result? 

cc @flaneur2020

Reply:
I do think so. 🤔

Reply:
my mistake。the method name should be get-retainable-x x x

Reply:
Fixed. Please review it again